scripts/globals/items/pinch_of_fanatics_powder.lua

```diff
@@ -0,0 +1,43 @@
+-----------------------------------------
+-- ID: 5847
+-- Item: pinch_of_fanatics_powder
+-- Item Effect: When applied, grants UDMGPHYS -10000 to party members in range for 60s
+-----------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    return 0
+end
+
+item_object.onItemUse = function(target)
+    local shieldtype = xi.effect.PHYSICAL_SHIELD
+    local duration = 60
+    local power = 1
+    local wsmitigation = 1
+
+    local function addShield(target, power, duration)
+        target:delStatusEffect(xi.effect.MAGIC_SHIELD)
+        target:addStatusEffect(shieldtype, power, 0, duration, 0, wsmitigation)
+        target:messageBasic(xi.msg.basic.GAINS_EFFECT_OF_STATUS, shieldtype)
+    end
+    
```

Comment:
whitespace here.
Also, it seems like the CI doesnt like something you are doing with target, power and duration in some scripts.

    scripts/globals/items/pinch_of_fanatics_powder.lua:21:30: (W432) shadowing upvalue argument target on line 15
    scripts/globals/items/pinch_of_fanatics_powder.lua:21:38: (W431) shadowing upvalue power on line 18
    scripts/globals/items/pinch_of_fanatics_powder.lua:21:45: (W431) shadowing upvalue duration on line 17